docs/content/s3.md

```diff
@@ -20,6 +20,7 @@ The S3 backend can be used with a number of different providers:
 {{< provider name="IBM COS S3" home="http://www.ibm.com/cloud/object-storage" config="/s3/#ibm-cos-s3" >}}
 {{< provider name="IDrive e2" home="https://www.idrive.com/e2/" config="/s3/#idrive-e2" >}}
 {{< provider name="IONOS Cloud" home="https://cloud.ionos.com/storage/object-storage" config="/s3/#ionos" >}}
+{{< provider name="Liara Object Storage (LOS)" home="https://liara.ir/landing/%D9%87%D8%A7%D8%B3%D8%AA-%D8%AF%D8%A7%D9%86%D9%84%D9%88%D8%AF-download-host" config="/s3/#liara-object-storage-los" >}}
```

Comment:
The home url here should be a regular human-readable web page, and I think the current is an API endpoint?